### PR TITLE
Typescripted and lintified version

### PR DESCRIPTION
**Pull request by dkuida**
Not sure if you are into that - but I migrated the version to typescript and added linting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,36 +1,70 @@
...
-    "node": ">=4"
+    "node": ">=8"
+  },
+  "publishConfig": {
+    "access": "public"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/jaakkos/winston-logstash.git"
+    "url": "git+https://github.com/dkuida/winston-logstash.git"
```
Comment (djMax) [inline package.json]: Probably should go back to previous value

Comment (dkuida) [reply package.json]: you are right - needed to use it asap - and wouldn't create a package with someone else name. removed

**Comment by dkuida** [comment]
Hey - any chance that you could merge it ? I o not want to maintain another fork reallyu

**Comment by dkuida** [comment]
@djMax  - coudl you review the PR 

it doesnt make sence to have a separate package for typescript

**Pull request closed without merging**